### DISCUSSION TITLE
Adding a quick note in Known Issues about VPN specific issues like MULLVAD

### DISCUSSION THREAD

**Discussion by aether-coder** [haugene/docker-transmission-openvpn#2622] [Ideas]
I am a beginner with dockers and I couldn't get haugene/transmission-openvpn to work with mullvad vpn until I did some searching around in the web, and like always the answer was reddit :) 
After a few threads I ended up in the following post:
https://www.reddit.com/r/synology/comments/r49gm4/need_help_with_transmission_container_in_docker/
I later also found the solution when I looked through the discussions and it pointed to the official documentation:
https://haugene.github.io/docker-transmission-openvpn/provider-specific/#mullvad_ovpn
I am temporarily using windows 11 to experiment with dockers and I  was able to get it working in power shell with the following command:

```
docker run --cap-add=NET_ADMIN -d --name=transmission-openvpn `
-v "C:\docker\transmission\data:/data" `
-v "C:\docker\transmission\config:/config" `
-e OPENVPN_PROVIDER=MULLVAD `
-e OPENVPN_CONFIG=us_chi `
-e OPENVPN_USERNAME=123456789101112(example) `
-e OPENVPN_PASSWORD=m `
-e LOCAL_NETWORK=192.168.1.0/24 `
--log-driver json-file `
--log-opt max-size=10m `
-p 9091:9091 `
--sysctl "net.ipv6.conf.all.disable_ipv6=0" `
--restart always `
haugene/transmission-openvpn

```

In conclusion, I just thought it would save a lot of time to some (specially beginners), if a quick note is added to the "Known Issues" section. That way they know beforehand there might be issues with their specific VPN.

**Comment by pkishino** [comment]
Huh? 
It’s in the documentation, anyone having issues should be looking there first
